docs/agent-vs-aea.md

```diff
@@ -20,8 +28,8 @@ from aea.helpers.file_io import write_with_lock
 from aea.identity.base import Identity
 from aea.mail.base import Envelope
 
-from packages.fetchai.connections.stub.connection import StubConnection
-from packages.fetchai.protocols.default.message import DefaultMessage
+from vendor.fetchai.connections.stub.connection import StubConnection
```

Comment:
Since this is an agent project, now we need to import from the vendor folder. This means that some tests need to be updated and account for this change.